sdk/spring/azure-spring-boot-starter-cosmos/README.md

```diff
@@ -76,13 +70,12 @@ Include actuator dependencies.
 </dependency>
 ```
 
-Call `http://{hostname}:{port}/actuator/health/cosmos` to get the CosmosDB health info. **Please note**: it will calculate [RUs](https://docs.microsoft.com/azure/cosmos-db/request-units).
+Call `http://{hostname}:{port}/actuator/health/cosmos` to get the Cosmos DB health info. **Please note**: it will calculate [RUs](https://docs.microsoft.com/azure/cosmos-db/request-units).
 
 ### Define an entity
 Define a simple entity as Document in Cosmos DB.
-<!-- embedme ../azure-spring-boot/src/samples/java/com/azure/spring/cosmosdb/User.java#L10-L65 -->
 ```java
```

Comment:
we need to add `<!-- embedme ../azure-spring-boot/src/samples/java/com/azure/spring/cosmosdb/User.java#L10-L65 -->` around code block, it helps us to check the code change

Reply:
oh, I got it wrong and thought it was the old path, OK, I will add back here

Reply:
@zhoufenqin  I also removed the description of properties supportting , `azure.cosmosdb.consistency-level` & `azure.cosmosdb.cosmosKeyCredential`, please help review.